docker/wordpress/Dockerfile

```diff
@@ -32,7 +32,7 @@ COPY install-plugin.sh /usr/local/bin/install-plugin
 RUN install-plugin wp-graphql wp-graphql/wp-graphql ${WPGRAPHQL_VERSION}
 
 # Install wp-gatsby using git, and apply a diff
-RUN git clone https://github.com/gatsbyjs/wp-gatsby ${PLUGIN_DIR}/wp-gatsby
+RUN git clone https://github.com/gatsbyjs/wp-gatsby ${PLUGIN_DIR}/wp-gatsby && git checkout ${WPGATSBY_VERSION}
```

Comment:
I think we need to cd into the directory before checking out the tag

Reply:
Or we can clone the tag directly https://stackoverflow.com/a/21699307/12777907

Reply:
--depth=1 is probably a good idea anyway :)

Reply:
i think i already resolved this, i just pushed the commit as you were reviewing! oops haha

Reply:
Perfect :D